tests/test_generator.py

```diff
@@ -0,0 +1,25 @@
+"""
+Tests for utility functions in the _generator.py module that cannot be
+exercised because of limitations in the Typer testing framework (specifically,
+multiple "prompt" arguments).
+"""
+from unittest import mock
+
+from pyscript import _generator as gen
+
+
+def test_create_project() -> None:
```

Comment:
I refactored this to not need mocks. Please let me know what you think. I couldn't figure out how to easily do a suggestion with a new file (`conftest.py`)